src/test/testing-library/SignUpText.test.tsx

```diff
@@ -1,10 +1,11 @@
 import { render } from '@testing-library/react';
+import { t } from 'i18next';
 
 import SignUpText from '../../features/landing/components/AuthSection/SignUpText/SignUpText';
 
-const socialTitle: string = 'Sign up now and free up your time to grow your business';
-const vilnaText: string = 'VilnaCRM';
-const socialsText: string = 'Log in with a convenient social network:';
+const socialTitle: string = t('sign_up.main_heading');
+const vilnaText: string = t('sign_up.vilna_text');
+const socialsText: string = t('sign_up.socials_main_heading');
```

Comment:
Please add the new test cases

Reply:
@igor-pereimyvovk we need to add as many tests as possible for both positive and negative test cases, so please add tests also for the content checking

Reply:
@Kravalg, if we mock the t function, we will lose our multilingual tests because we would have to hardcode the text in a specific language. And if we take the text for the mock from the localization file, it would just be dead code. It would be a step backward. I don't think this will improve the system. I also want to point out that with the help of stryker, we are literally testing all the text on the page, including different aria-labels.

Reply:
@igor-pereimyvovk, we need positive and negative test cases to thoroughly test the translation system itself. We can probably add tests for the t function to verify that it returns the correct translation for a given key. That should be sufficient for our needs.